mmv1/third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -1167,6 +1225,99 @@ func flattenSecurityPolicyRedirectOptions(conf *compute.SecurityPolicyRuleRedire
 	return []map[string]interface{}{data}
 }
 
+func expandSecurityPolicyRecaptchaOptionsConfig(configured []interface{}, d *schema.ResourceData) *compute.SecurityPolicyRecaptchaOptionsConfig {
+	if len(configured) == 0 || configured[0] == nil {
+		return nil
+	}
+
+	data := configured[0].(map[string]interface{})
+
+	return &compute.SecurityPolicyRecaptchaOptionsConfig{
+		RedirectSiteKey: data["redirect_site_key"].(string),
+		ForceSendFields: []string{"RedirectSiteKey"},
```

Comment:
Is `ForceSendFields` needed to support `""`, to allow for reverting to the Google-managed site key? I saw this in one of the tests, but wanted to confirm I understood correctly.

Reply:
Yes, that is correct. The doc mentions that the user can also disassociate a reCAPTCHA WAF site key with a security policy as follows by setting RedirectSiteKey to "".